flocker/ca/_script.py

```diff
@@ -90,6 +90,56 @@ def getSynopsis(self):
 
 
 @flocker_standard_options
+class NodeCertificateOptions(PrettyOptions):
+    """
+    Command line options for ``flocker-ca create-node-certificate``.
+    """
+
+    helptext = """Create a new certificate for a node agent.
+
+    Creates a certificate signed by a previously generated certificate
+    authority (see flocker-ca initialize command for more information).
+    """
+
+    synopsis = "[options]"
+
+    optParameters = [
+        ['inputpath', 'i', os.getcwd(),
+         'Path to directory containing root certificate.'],
+        ['outputpath', 'o', os.getcwd(),
+         'Path to directory to write control service certificate.'],
```

Comment:
It is unusual to have default arguments that change with context. If I saw the default value once in the usage message, I would expect it to stay as that value every time I ran the command. I think instances of this should display that the default is the current working directory, not the value of the current working directory at this moment.